packages/block-editor/src/components/link-control/index.js

```diff
@@ -134,6 +141,19 @@ function LinkControl( {
 	hasTextControl = false,
 	renderControlBottom = null,
 } ) {
+	if ( hasNofollowSetting ) {
+		settings = [
+			...settings,
+			{
+				id: 'nofollow',
+				title: createInterpolateElement(
+					__( 'Mark as <code>nofollow</code>' ),
```

Comment:
This causes a screen reader accessibility issue. No code tags in labels. Not actually sure why this is as it seems to be allowed per the HTML specification but with testing, it is an issue.

Maybe this is a better label?

```suggestion
					__( 'Add no-follow for search engines' ),
```

Reply:
> This causes a screen reader accessibility issue. No code tags in labels

Good to know! I am okay with the alternative label, so let's ask for other's thoughts eg: @scruffian, then I will update the e2e testing for the new labels

Reply:
Makes sense to me.

Reply:
Addressed on the latest commit, I also updated the e2e testing for selecting the new label
<img width="356" alt="image" src="https://github.com/WordPress/gutenberg/assets/10071857/2e5cd2b1-a7f7-4e8e-bae8-3a1496425170">

Also I had another [question here](https://github.com/WordPress/gutenberg/pull/54110#discussion_r1312995012), do you mind to take a look as well @alexstine? Thanks

Reply:
@bangank36 Thanks. I honestly don't know about your other question. I mainly focus on UI accessibility in Gutenberg, the internals of managing settings is an unknown area to me. It sounds like it makes sense but that's really all I can offer.

Reply:
This will need updating in `trunk` for Rich Text as well.

Reply:
You mean for the [inline.js](https://github.com/WordPress/gutenberg/blob/ab5a41c7979b81f0827289f1adde7f505bcc2702/packages/format-library/src/link/inline.js#L34) file? 
https://github.com/WordPress/gutenberg/blob/8ae150433d58ffbb0536f4bc1435b043b905b0a0/packages/format-library/src/link/inline.js#L38

Reply:
Yes. I can't recall whether I used a `code` tag there or not in the end.

If I did then we'll need a separate PR to fix that one.

Reply:
See https://github.com/WordPress/gutenberg/issues/54179

Reply:
In https://github.com/WordPress/gutenberg/pull/53945 you used `<code>` tag, and I changed the inline.js and the e2e testings for link with the suggested label already ( along with the button link changes )
If you think they need to be on its own PR then I will submit a new commit to remove these changes from this PR.
Also I may need your thoughts on https://github.com/WordPress/gutenberg/pull/54110#discussion_r1312995012

Thanks

Reply:
> If you think they need to be on its own PR then I will submit a new commit to remove these changes from this PR.

Whilst it seems over the top, I would make a separate PR. Why? Because if _this_ PR gets rejected or takes a while to land the a11y bug will continue to exist in `trunk`. With a separate PR we can quickly and easily revert the issue immediately. It's also a better separate of concerns.

Given that you have the code already, would you be open to raising that PR?

Reply:
On it!

Reply:
See https://github.com/WordPress/gutenberg/pull/54184